### PR TITLE
Ver 76907: Verify JDBC SSL params for connector

### PR DESCRIPTION
**Pull request by jonathanl-bq**
### Summary

This PR replaces the the ssl connector option with the newer, not deprecated tls_mode option.

### Description

This change updates the connector options as well as the documentation.

### Related Issue

http://jira.verticacorp.com:8080/jira/browse/VER-76907

### Additional Reviewers

@alexr-bq 
@NerdLogic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,10 @@ Here is an example spark-submit command showing how to add Spark's default confi
 `spark-submit --master spark://spark:7077 --conf "spark.driver.extraClassPath={$SPARK_HOME}/conf/" --deploy-mode cluster app-assembly.jar --class Main`
 where `{$SPARK_HOME}` is the `$SPARK_HOME` directory on the worker node.
 
+## Additional guides
+
+For information on how to configure Kerberos and TLS with the connector, see the KerberosUserManual.md and TLSConfiguration.md files in the root of this repository.
```
Comment (alexey-temnikov) [inline README.md:136]: Can we have  KerberosUserManual.md and TLSConfiguration.md  as links?

Comment (jonathanl-bq) [reply README.md:136]: I'm not sure; They live at the root level alongside this README. We could add folders for them with nothing else in them. Would that be okay?

Comment (alexr-bq) [reply README.md:136]: You can link direct 

See: Guide on setting up a single-node HDFS

**Review by alexr-bq — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jonathanl-bq) [reply README.md]: :O fixed

**Pull request merged**